### ISSUE TITLE
intelmqctl log does not work without system.conf

### ISSUE THREAD

**Issue by bernhardreiter** [certtools/intelmq#754]
See https://github.com/certtools/intelmq/blob/master/intelmq/bin/intelmqctl.py#L571
the system configuration is still used to configure the logging path.

 intelmqctl log postgresql-output
will give something like

```
File "/usr/lib/python3/dist-packages/intelmq/bin/intelmqctl.py", line 571, in read_bot_log
    bot_log_path = os.path.join(self.system['logging_path'],
AttributeError: 'IntelMQContoller' object has no attribute 'system'
```

**Comment by sebix** [comment]
Yeah, I know. It's fixed in #719 and I was lazy, not cherry-picking it as I hoped the PR would be reviewed and merged soon anyway.

https://github.com/certtools/intelmq/pull/719/files#diff-830172912b05cf53aa0c3e766d1ceac6L571

**Comment by sebix** [comment]
Fixed in master.
